lib/modules/platform/gerrit/index.md

```diff
@@ -0,0 +1,72 @@
+# Gerrit
+
+## Supported Gerrit Versions
+
+All Gerrit 3.x Versions should be working.
+
+The current implementation uses the "hashtags" Feature from Gerrit and therefore needs a Gerrit version with [NoteDB](https://gerrit-review.googlesource.com/Documentation/note-db.html) backend.
+Gerrit `2.x` with NoteDB (only in `2.15` and `2.16`) is not tested, but could work.
+
+## Authentication
+
+<figure markdown>
+  ![Gerrit HTTP access token](/assets/images/gerrit-http-password.png){ loading=lazy }
+  <figcaption>First, create a HTTP access token for the Renovate account.</figcaption>
+</figure>
+
+Let Renovate use your HTTP access token by doing _one_ of the following:
+
+- Set your HTTP access token as a `password` in your `config.js` file
+- Set your HTTP access token as an environment variable `RENOVATE_PASSWORD`
+- Set your HTTP access token when you run Renovate in the CLI with `--password=`
+
+This user must be allowed to assign the Code-Review label with "+2" to their own changes for "automerge" to work.
```

Comment:
It's not clear to me what "This user" refers to. I think we mean "The Gerrit user account that Renovate will work through", or something like that?

Reply:
@HonkingGoose Perhaps we should mention that the username parameter is needed too? 
I'm not sure if this is a general requirement for all/other platforms supported by Renovate.